Sources/ParseSwift/Types/ParseError.swift

```diff
@@ -14,6 +14,12 @@ import Foundation
 public struct ParseError: ParseType, Decodable, Swift.Error {
     public let code: Code
     public let message: String
+    public var intCode: Int? = nil
+    init(code: Code, message: String) {
```

Comment:
I’m on my phone, but probably will need to make the init public for devs to use it outside of the SDK